3dVolViewer adds (#402)
DGtal-team/DGtalTools: ChangeLog.md

# DGtalTools 1.2
- *global*

>   - Fix itk2vol and fix ITK cmake configuaration that was making wrong the ITK image read.

Reviewer: ```suggestion
  - Fix itk2vol and fix ITK cmake configuaration that was making issues with the ITK image read.
```

    (Bertrand Kerautret [#393](https://github.com/DGtal-team/DGtalTools/pull/393))
  - Travis: Fix old default osx_image with xcode12.2 and remove non used boost
    cmake references. (Bertrand Kerautret [#394](https://github.com/DGtal-team/DGtalTools/pull/394)) 

- *visualisation*:

>   - 3dVolViewer: improvement with the possibility to read input image of type double (with ITK).

Reviewer: ```suggestion
  - 3dVolViewer: improvement of the possibility to read input image of type double (with ITK).
```

>     New possibility to selection the voxel in order to display image intensity.    

Reviewer: ```suggestion
    New possibility to select the voxel in order to display image intensity.    
```
